pdocs/doc.py

```diff
@@ -148,6 +150,14 @@ def __init__(self, name, module, docstring):
         by `inspect.cleandoc`.
         """
 
+        try:
+            self.parsed_docsting = docstring_parser.parse(self.docstring)
```

Comment:
spelling error: should be 
```suggestion
            self.parsed_docstring = docstring_parser.parse(self.docstring)
```